.github/workflows/release.yml

```diff
@@ -0,0 +1,32 @@
+# .github/workflows/release.yml
+
+on:
+  release:
+    types: [created]
```

Comment:
We should probably do something like this instead: https://github.com/Screenly/Anthias/blob/master/.github/workflows/build-balena-disk-image.yaml#L2-L5